src/App.js

```diff
@@ -1,86 +1,201 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { TodoList } from './components/TodoList';
+import { Filters } from './components/Filters';
+
+const useLocalStorage = (key, initialValue) => {
+  const [value, setValue] = useState(
+    JSON.parse(localStorage.getItem(key)) || initialValue,
+  );
+  const save = (newValue) => {
+    setValue(newValue);
+    localStorage.setItem(key, JSON.stringify(newValue));
+  };
+
+  return [value, save];
+};
 
 function App() {
+  const [title, setTitle] = useState('');
+  const [todos, setTodos] = useLocalStorage('todos', []);
+  const [filteredTodos, setfilteredTodos] = useState(todos);
+  const [filterStatus, setFilterStatus] = useState('');
```

Comment:
let's make the default filter just 'all' and there will be no need to check for an empty string later

```suggestion
  const [filterStatus, setFilterStatus] = useState('all');
```